com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/transport/ZigBeePort.java

```diff
@@ -85,6 +85,16 @@ public interface ZigBeePort {
      */
     void purgeRxBuffer();
 
+    /**
```

Comment:
These methods should not be added here. This interface is not specific to a serial port - that is implemented in the `ZigBeeSerialPort` class. Here we need to keep it more abstract as there are other implementations of this such as an SPI implementation.